pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -50,6 +50,7 @@
 , withIconv ? withHeadlessDeps
 , withJack ? withFullDeps && !stdenv.isDarwin # Jack audio
 , withLadspa ? withFullDeps # LADSPA audio filtering
+, withLibaribcaption ? withFullDeps # Portable ARIB STD-B24 Caption Decoder/Renderer
```

```suggestion
, withLibaribcaption ? withFullDeps && versionAtLeast version "6.1" # Portable ARIB STD-B24 Caption Decoder/Renderer
```

to avoid adding libaribcaption to buildInputs for ffmpeg 6.0

---

Done.